perl/lib/Sanger/CGP/Pindel/Implement.pm

```diff
@@ -333,20 +333,25 @@ sub valid_seqs {
   while(<$FAI_IN>) { push @all_seqs, (split /\t/, $_)[0]; }
   close $FAI_IN;
 
+  my @exclude;
+  my @exclude_patt = [];
```

Comment:
I'm doubtful about this. You're going to exclude sequences which match `ARRAY(0x563dea4d1430)` or similar,

Reply:
ce81a0e Now create empty arrays